tests/foreman/cli/test_contentview.py

```diff
@@ -885,9 +884,6 @@ def test_negative_add_same_puppet_repo_twice(self):
 
         @assert: User cannot add puppet modules multiple times to the content
         view
-
-        @bz: 1222118
-
         """
```

Comment:
When we remove bz references, may be we need to start commenting something like this in the first line of tests.

```
# see BZ #1222118
repository = make_repository({
       u'content-type': u'puppet',
       u'product-id': self.product['id'],
       u'url': FAKE_0_PUPPET_REPO,
})
```
this will help us for future debugging.